doc/nvim-tree-lua.txt

```diff
@@ -1062,7 +1066,8 @@ DEFAULT MAPPINGS                                     *nvim-tree-default-mappings
     { key = "]c",                             action = "next_git_item" }
     { key = "-",                              action = "dir_up" }
     { key = "s",                              action = "system_open" }
-    { key = "f",                              action = "live_filter" }
+    { key = "ff",                             action = "live_filter" }
```

Comment:
On pressing ff when fb is active, the filter changes text to `/Buffer/` which is not useful.